core/templates/pages/admin-page/activities-tab/admin-dev-mode-activities-tab.directive.html

```diff
@@ -30,19 +30,21 @@ <h3>Generate dummy explorations</h3>
       Use this form to generate dummy explorations with random titles.
     </p>
     <span class="row">
-      <label for="explorations to generate">Number of explorations to generate</label>
+      <label for="explorations-to-generate">Number of explorations to generate</label>
       <input class="form-control form-inline" style="margin-bottom: 5px"
              ng-model="$ctrl.numDummyExpsToGenerate"
-             placeholder="Explorations to generate" type="number" min="0">
-      <label for="explorations to publish">Number of explorations to publish</label>
+             placeholder="Explorations to generate" type="number" min="0"
+             id = "explorations-to-generate">
+      <label for="explorations-to-publish">Number of explorations to publish</label>
       <em ng-show="$ctrl.numDummyExpsToGenerate"
           style="font-size: smaller;">
         (Publish <[$ctrl.numDummyExpsToPublish]> out of <[$ctrl.numDummyExpsToGenerate]> explorations)
       </em>
       <input class="form-control form-inline"
              ng-model="$ctrl.numDummyExpsToPublish"
              placeholder="Explorations to Publish"
-             ng-disabled="!$ctrl.numDummyExpsToGenerate" type="number" min="0" >
+             ng-disabled="!$ctrl.numDummyExpsToGenerate" type="number" min="0"
+             id = "explorations-to-publish">
```

Comment:
Why do we need an id here? (Also, remove the space before and after `=`)

Reply:
We need the id attribute because we need to connect form elements to labels. 
https://web.dev/label/?utm_source=lighthouse&utm_medium=devtools

the `id` attribute in the `input` tag connects to the `for` attribute in the `label` tag. When a screen reader goes over the input box, the label will be announced to them. In this case, the screen reader will read "Number of explorations to publish".

Reply:
Ah, I see. I misunderstood the usage there! Thanks for the explanation!